src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
```

Comment:
make it not final. Same in anothe cases

Reply:
ok, I did it at first but then receved warning from IDEA and made it 'final'